src/main/java/seedu/fast/logic/parser/FindCommandParser.java

```diff
@@ -24,10 +26,33 @@ public FindCommand parse(String args) throws ParseException {
             throw new ParseException(
                     String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
         }
+        if (trimmedArgs.startsWith(PriorityTag.PRIORITY_TAG_PREFIX)) {
+            trimmedArgs = trimmedArgs.substring(
+                    PriorityTag.PRIORITY_TAG_PREFIX.length());
+            String[] tags = trimmedArgs.split("\\s+");
```

Comment:
I think you can include a comment as to what this split does to make it more understandable